Sources/SQLiteNIO/SQLiteData.swift

```diff
@@ -90,7 +90,7 @@ public enum SQLiteData: Equatable, Encodable, CustomStringConvertible {
         case .float(let float): return float.description
         case .integer(let int): return int.description
         case .null: return "null"
-        case .text(let text): return "\"" + text + "\""
+        case .text(let text): return #""\#(text)""#
```

Comment:
glad you changed this code .... i was getting dizzy trying to notice what was going on in the previous code.

Reply:
Amen to that.